app/src/main/java/kth/jjve/xfran/WorkoutsTabActivity.java

```diff
@@ -103,4 +99,11 @@ private void initNavMenu(){
         navigationView.setNavigationItemSelectedListener(this);
         navigationView.setCheckedItem(R.id.nav_home);
     }
+
+    @Override
+    public void onListItemClick(int position) {
```

Comment:
I only see commented stuff here. Instead of commenting you could just take it away (version control). Furthermore, I would like to have a comment here what this method should do

Reply:
Since we use version control. You can just take away line 111 and 113. What the function should do (line 112) can stay